snapcraft/meta.py

```diff
@@ -202,7 +200,7 @@ def _write_wrap_exe(wrapexec, wrappath, shebang=None, args=[], cwd=None):
     os.chmod(wrappath, 0o755)
 
 
-def _wrap_exe(relexepath):
+def _wrap_exe(relexepath, args=[]):
```

Comment:
it's generally wrong to define a mutable structure as a default argument.

Take a look here:
http://effbot.org/zone/default-values.htm

What you should do is 
def...(...,args=None):
   if args is None:
       args = []